### PR TITLE
[client] Split the close icon into single svg file.

### PR DESCRIPTION
**Pull request by saneyuki**
By Content Security Policy spec,
http://w3c.github.io/webappsec/specs/content-security-policy/#source-list-guid-matching

`data:` schemes cannot be loaded in the `*` policy.

[Review on Reviewable](https://reviewable.io/reviews/karen-irc/karen/301)

### CODE REVIEW

File: client/img/close-button.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
+ <g>
+  <title>Layer 1</title>
```
Comment (saneyuki) [inline client/img/close-button.svg]: fix

**Comment by saneyuki** [comment]
@dokidokivisual r+

**Comment by dokidokivisual** [comment]
:pushpin: Commit 92b8f41 has been approved by `saneyuki`

**Comment by dokidokivisual** [comment]
:hourglass: Testing commit 92b8f41 with merge c7f69e2...

**Comment by dokidokivisual** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/karen-irc/karen/builds/74538026)

**Pull request merged**